data/component-design-tokens/filter-chip-design-tokens.json

```
@@ -0,0 +1,127 @@
+      "width": "{borderWidth.s}",
+      "style": "solid"
+    },
+    "type": "border"
+  },
+  "filter-chip-border-color-hover": {
+    "value": "{color.greyscale.500}",
+    "type": "color"
+  },
+  "filter-chip-text-color": {
+    "value": "{color.text.default}",
+    "type": "color"
+  },
+  "filter-chip-color-bg": {
+    "value": "{color.greyscale.white}",
+    "type": "color"
+  },
+  "filter-chip-color-bg-hover": {
+    "value": "{color.greyscale.100}",
+    "type": "color"
+  },
+  "filter-chip-text-color-error": {
+    "value": "{input.color.border.error}",
+    "type": "color"
+  },
+  "filter-chip-text-color-error-hover": {
+    "value": "{input.color.text.error}",
+    "type": "color"
+  },
+  "filter-chip-color-bg-error": {
+    "value": "{input.color.background.error}",
+    "type": "color"
+  },
+  "filter-chip-color-bg-error-hover": {
+    "value": "{input.color.background.error-hover}",
+    "type": "color"
+  },
+  "filter-chip-border-color-error": {
+    "value": "{input.color.border.error}",
+    "type": "color"
+  },
+  "filter-chip-border-color-error-hover": {
+    "value": "{input.color.border.error-hover}",
+    "type": "color"
+  },
+  "filter-chip-icon-color-error": {
+    "value": "{input.color.border.error}",
+    "type": "color"
+  },
+  "filter-chip-icon-color-error-hover": {
+    "value": "{input.color.text.error}",
+    "type": "color"
+  },
+  "filter-chip-leading-icon-color": {
+    "value": "{color.greyscale.600}",
+    "type": "color"
+  },
+  "filter-chip-secondary-text-color": {
+    "value": "{color.greyscale.600}",
+    "type": "color"
+  },
+  "filter-chip-padding-vertical": {
+    "value": "5px",
+    "type": "spacing"
+  },
+  "filter-chip-padding-horizontal-left": {
+    "value": "{space.s}",
+    "type": "spacing"
+  },
+  "filter-chip-padding-horizontal-right": {
+    "value": "{space.xs}",
+    "type": "spacing"
+  },
+  "filter-chip-gap": {
+    "value": "{space.s}",
+    "type": "spacing"
+  },
+  "filter-chip-label-gap": {
+    "value": "6px",
+    "type": "spacing"
+  },
+  "filter-chip-typography": {
+    "value": "{typography.body.s}",
+    "type": "other"
+  },
+  "filter-chip-line-height": {
+    "value": "{lineHeight.2}",
+    "type": "lineHeights"
+  },
+  "filter-chip-border-focus-width": {
+    "value": "{borderWidth.l}",
+    "type": "borderWidth"
+  },
+  "filter-chip-border-focus-offset": {
+    "value": "{space.3xs}",
+    "type": "sizing",
+    "description": "2px"
+  },
+  "filter-chip-border-focus-color": {
+    "value": "{color.interactive.focus}",
+    "type": "color"
+  },
+  "filter-chip-min-height": {
+    "value": "{space.xl}",
+    "type": "sizing"
+  },
+  "filter-chip-min-width": {
+    "value": "56px",
+    "type": "sizing"
+  },
+  "filter-chip-icon-color-default": {
+    "value": "{color.greyscale.700}",
+    "type": "color"
+  },
+  "filter-chip-icon-color-hover": {
+    "value": "{color.greyscale.black}",
+    "type": "color"
+  }
```

```suggestion
  },
  "filter-chip-secondary-text-color-error": {
    "value": "{input.color.text.error}",
    "type": "color"
  },
  "filter-chip-close-button-error-hover-bg-color": {
    "value": "{input.color.background.error-hover}",
    "type": "color"
  }
```

- The CSS references this token for secondary text in error state, but it didn't exist. Uses input.color.text.error to match the leading icon.
- The CSS passes this to the icon button to style its hover state in error mode. Without it, the icon button's destructive hover background would be undefined.